src/hotspot/cpu/x86/stubGenerator_x86_64_fmod.cpp

```diff
@@ -0,0 +1,546 @@
+/*
+ * Copyright (c) 2016, 2021, 2023, Intel Corporation. All rights reserved.
```

Comment:
This is a new file. Copyright (c)  2023, Intel Corporation. All rights reserved.

Reply:
Done.